README.md

```diff
@@ -9,22 +9,19 @@ products:
 - redux
 - dotnet
 - azure-active-directory
+- msal

```

Comment:
```suggestion
- msal
- msal-browser
```

Reply:
this section is mostly for ms docs indexing the sample. will keep it "msal" for better results.